src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,133 @@
+package lotto.controller;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import lotto.domain.LottoPrice;
+import lotto.mapper.LottoMapper;
+import lotto.domain.util.LottoNumberParser;
+import lotto.domain.rank.LottoRanks;
+import lotto.domain.Number;
+import lotto.domain.lotto.WinningLotto;
+import lotto.domain.lotto.random.CreateRandomNumbers;
+import lotto.domain.lotto.Lotto;
+import lotto.domain.lotto.Lottos;
+import lotto.domain.Money;
+import lotto.domain.util.LottoStatistics;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+import lotto.view.dto.LottoCountResponse;
+import lotto.view.dto.LottoErrorResponse;
+import lotto.view.dto.LottoRankResponse;
+import lotto.view.dto.LottoResponse;
+import lotto.view.dto.ReturnRateResponse;
+
+public class LottoController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final CreateRandomNumbers createRandomNumbers;
+
+    public LottoController(InputView inputView, OutputView outputView, CreateRandomNumbers createRandomNumbers) {
```

Comment:
```suggestion
    public LottoController(InputView inputView, OutputView outputView, RandomNumbersCreator randomNumbersCreator) {
```
클래스 이름은 행위가 아닌 명사형으로 만드는건 어떨까요

Reply:
명사형으로 하니, 훨씬 와닿는 것 같아요 좋은 말씀 감사합니다!